static/app/components/feedback/feedbackItem/useUpdateFeedback.tsx

```diff
@@ -24,20 +25,25 @@ export default function useResolveFeedback({feedbackItem}: Props) {
     return `/organizations/${organization.slug}/issues/${feedbackId}/`;
   }, [feedbackId, organization]);
 
-  const handleResolve = useCallback(async () => {
-    addLoadingMessage(t('Updating feedback...'));
-    try {
-      await api.requestPromise(url, {
-        method: 'PUT',
-        data: {status: feedbackItem.status === 'unresolved' ? 'resolved' : 'unresolved'},
-      });
-      addSuccessMessage(t('Updated feedback'));
-    } catch {
-      addErrorMessage(t('An error occurred while resolving the feedback.'));
-    }
-  }, [api, url, feedbackItem.status]);
+  const handleUpdate = useCallback(
+    async (newStatus: GroupStatus) => {
+      addLoadingMessage(t('Updating feedback...'));
+      try {
+        await api.requestPromise(url, {
+          method: 'PUT',
+          data: {
+            status: newStatus,
+          },
+        });
+        addSuccessMessage(t('Updated feedback'));
+      } catch {
+        addErrorMessage(t('An error occurred while updating the feedback.'));
+      }
+    },
+    [api, url]
+  );
 
   return {
-    onResolve: () => handleResolve(),
+    onUpdate: (newStatus: GroupStatus) => handleUpdate(newStatus),
```

Comment:
nit: i'd rename this to be 'onSetStatus' or something so it's clear in all the spots (within the other file mainly) what is being updated.